rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search/100_stored_fields.yml

```diff
@@ -40,3 +40,19 @@ setup:
 
   - is_false: hits.hits.0._id
   - is_false: hits.hits.0._source
+
+---
+"Stored fields - Support _none_ wrapped in array":
+  - skip:
+      version: " - 8.12.99"
+      reason: This was raising an NPE until 8.13.0
```

Comment:
Is this not testable outside of yaml tests? I am mentioning this because the skip is slightly odd, but not a huge deal either.

Reply:
Yeap - now that this has been added as a simple unit tests we can remove this one.